DOCS-6735 ACM: Dec 7 release (#122)
mulesoft/docs-api-community-manager: modules/ROOT/pages/browser-support.adoc

= Browser Support
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images/

> Community admins, community members, and guest users can access an API Community Manager community by using the latest versions of these browsers:

Reviewer: Consider spelling out "administrator," as is done in the intro to the following topic.
Are we distinguishing between and administrator in general and an "admin" user?
Check globally.

Author: We are not distinguishing between those. Changed to "administrator".

> 

Reviewer: Is the following list in any audience-discernable order?
If not, consider alpha.

Author: Alpha is better-- switched to that.

* Google Chrome
* Mozilla Firefox
* Apple Safari
* Microsoft Edge
* Microsoft Internet Explorer 11 (limited)
+

> Microsoft Internet Explorer 11 has limited compatibility, and will display content-only, non-interactive components, including the API Carousel, API Catalog, and API Documentation Viewer. The components API Console, API Access Requester, and My Applications are not supported in Microsoft Internet Explorer 11, and will render in a disabled state with a message to the user explaining the compatibility limitations.

Reviewer: ```suggestion
Microsoft Internet Explorer 11 compatibility is limited to diplaying content-only, non-interactive components, including API Carousel, API Catalog, and API Documentation Viewer. The components API Console, API Access Requester, and My Applications are not supported in Microsoft Internet Explorer 11, and render in a disabled state with a message to the user explaining the compatibility limitations.
```
